lib/dialects/postgres/connection-manager.js

```diff
@@ -109,18 +110,21 @@ class ConnectionManager extends AbstractConnectionManager {
           return;
         }
         responded = true;
+        debug(`connection acquire`);
```

Comment:
connection object don't have `UUID` when I try to access it. It always comes `undefined`. I was planning to add it but I don't know which parameter should I use other than `UUID`